src/app.js

```diff
@@ -82,11 +82,11 @@ app.use((req, res, next) => {
   logger.info(`[${req.method}] ${req.originalUrl}`);
   next();
 });
-app.use("/api/v1/user", UserRouter);
+app.use("/api/v1/auth/user", UserRouter);
```

Comment:
You've only updated the endpoint but the functionality is still in the user controller. This update would be appropriate if the there was an auth router and auth controller.